src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -41,60 +37,68 @@ public enum armPosition {
     // braking)
     this.rightMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
-    this.rightMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, FORWARD_LIMIT);
+    this.rightMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, Constants.ArmConstants.FORWARD_LIMIT);
     this.rightMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
-    this.rightMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, REVERSE_LIMIT);
+    this.rightMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse,  Constants.ArmConstants.REVERSE_LIMIT);
 
     // setting the idle mode setting for the SparkMax, here it is set to brake when idle
     this.rightMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
 
-    // motor type for left motor
-    this.leftMotor = new CANSparkMax(leftMotorCANID, MotorType.kBrushless);
+    this.rightMotor.setInverted(Constants.ArmConstants.RIGHT_MOTOR_INVERTED);
 
-    // comment below is for incase the motors need to be reversed
-    // this.leftMotor.setInverted(invertLeftMotor);
+    this.leftMotor.follow(this.rightMotor, Constants.ArmConstants.LEFT_MOTOR_INVERTED);
 
     this.leftMotor.follow(this.rightMotor, true);
 
-    armPositions.put(armPosition.INTAKE_ARM_POSITION_GROUND, INTAKE_ARM_GROUND);
-    armPositions.put(armPosition.INTAKE_ARM_POSITION_SHELF, INTAKE_ARM_SHELF);
-    armPositions.put(armPosition.SCORING_ARM_POSITION_LOW, SCORING_ARM_LOW);
-    armPositions.put(armPosition.SCORING_ARM_POSITION_MID, SCORING_ARM_MID);
+    //leftmotor.burnflash
+    //rightmotor.burnflash
+
+    armPositions.put(armPosition.INTAKE_ARM_POSITION_GROUND,  Constants.ArmConstants.INTAKE_ARM_GROUND);
+    armPositions.put(armPosition.INTAKE_ARM_POSITION_SHELF,  Constants.ArmConstants.INTAKE_ARM_SHELF);
+    armPositions.put(armPosition.SCORING_ARM_POSITION_LOW,  Constants.ArmConstants.SCORING_ARM_LOW);
+    armPositions.put(armPosition.SCORING_ARM_POSITION_MID, Constants.ArmConstants. SCORING_ARM_MID);
   }
 
   // setting functions of runToScore and runToIntake
   public void runToScoreLow() {
-    this.rightMotor.set(FORWARD_SPEED);
+    this.rightMotor.set( Constants.ArmConstants.FORWARD_SPEED);
     this.currentArmPosition = armPosition.SCORING_ARM_POSITION_LOW;
   }
 
   public void runToScoreMid() {
-    this.rightMotor.set(FORWARD_SPEED);
+    this.rightMotor.set( Constants.ArmConstants.FORWARD_SPEED);
     this.currentArmPosition = armPosition.SCORING_ARM_POSITION_MID;
   }
 
   public void runToIntakeGround() {
-    this.rightMotor.set(REVERSE_SPEED);
+    this.rightMotor.set( Constants.ArmConstants.REVERSE_SPEED);
     this.currentArmPosition = armPosition.INTAKE_ARM_POSITION_GROUND;
   }
 
   public void runToIntakeShelf() {
-    this.rightMotor.set(REVERSE_SPEED);
+    this.rightMotor.set( Constants.ArmConstants.REVERSE_SPEED);
     this.currentArmPosition = armPosition.INTAKE_ARM_POSITION_SHELF;
   }
```

Comment:
These two functions are different only by the position. See if you can combine these functions into a single function that takes the position as an argument.